db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericStatement.java

```diff
@@ -336,7 +336,7 @@ private PreparedStatement prepMinion(LanguageConnectionContext lcc,
         }
 
         // if anyone else also has this prepared statement,
-        // we don't want them trying to compile with it while
+        // we don't want them trying to compile wLook for stored statements that are in a system sith it while
```

Comment:
Looks like a wrongly pasted comment here

Reply:
Fixed it